### ISSUE TITLE
Stud texture issue on zoom out

### ISSUE THREAD

**Issue by Symbian9** [leozide/leocad#96]
Studs look ugly on some distance if zoom out scene.

![Stud texture issue](http://hnng.moe/f/T2R)

**Version**: OBS build (built on 31Jul 2017)

**OS**: xUbuntu 16.04 LTS, x86_64

**Comment by leozide** [comment]
I've never seen this before, it's probably your video card driver. Have you tried updating?

**Comment by Symbian9** [comment]
When I use zoom its look like on some zoom level one texture switched to another one.

Is it how this stud texture work in LeoCAD?

Is it possible add vector (non-bitmap) stud texture mode?

**Comment by leozide** [comment]
This is likely a bug in your drivers, can you post the contents of the About Dialog?

**Comment by Symbian9** [comment]
@leozide, here is screen from my desktop

![info](http://hnng.moe/f/Tvy)

OS: Linux Mint 18.1 (xUbuntu 16.04 LTS)

GPU: ATI Radeon Xpress 1250 (integrated)

**Comment by Symbian9** [comment]
For the record, why not use vector studs in LeoCAD?

**Comment by leozide** [comment]
See if f35a9ff1b64f5c37c484dd7e4105c48afdb1b71f helps. You're the only person who has this problem.

A texture is better than geometry in this case because it alias less and is faster.

**Comment by Symbian9** [comment]
> See if `f35a9ff` helps.

@leozide, could you provide atleast 'nightly' builds as AppImage, because current OBS builds look like broken
* https://github.com/leozide/leocad/issues/111

**Comment by leozide** [comment]
You can download the Travis builds, check the bottom of the build log for the link.

**Comment by Symbian9** [comment]
> See if `f35a9ff` helps.

No, still look like two different textures on some zoom level switched from on le to another one

**Comment by j6t** [comment]
Is this still an issue? (I can't tell what the issue actually is because the screenshot links are dead.)

**Comment by Symbian9** [comment]
> Is this still an issue? (I can't tell what the issue actually is because the screenshot links are dead.)

@j6t, Yes. (I just updated screenshots taken with `LeoCAD-Linux-278e953-x86_64.AppImage`)
